modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_a11y.js

```diff
@@ -15,11 +15,14 @@
 AUI.add(
 	'liferay-calendar-a11y',
 	(A) => {
+		A.CalendarBase.ATTRS.tabIndex.value = 0;
+
 		let template = A.CalendarBase.HEADER_TEMPLATE;
 
-		template = template.replace('aria-role="heading"', '');
-		template = template.replace('<div class="yui3-u', '<h1 class="yui3-u');
-		template = template.replace(/div></, 'h1><');
+		template = template.replace(
```

Comment:
Could we completely remove role heading?

The client is complying that H1 element could be a problem if on the page we have another H1 element.

Reply:
TBH, I'm not too sure when it comes to the a11y stuff...

Maybe @pat270 knows a bit more on that?